HIRS_Utils/src/main/java/hirs/data/persist/certificate/EndorsementCredential.java

```diff
@@ -373,6 +373,8 @@ private void parseSequence(final ASN1Sequence seq, final boolean addToMapping,
                     revision.getValue());
             LOGGER.debug("Found TPM Spec:" + tpmSpecification.toString());
         } else if (addToMapping && key.equals(TPM_SECURITY_ASSERTIONS)) {
+            // TODO(apldev3): Update this block to properly parse TPM Security Assertions
```

Comment:
Ignore any TODOs you see here, this is taken care of in the next MR that will be put up after this one.